README.md

```diff
@@ -77,45 +79,49 @@ komac update
-| Progress bar                             |      ✅     |       ❌      |                              ❌                             |
-| Language                                 | Kotlin/JVM |      C#      |                         PowerShell                         |
+While other manifest creation tools have made remarkable strides in the winget-pkgs community, their development pace is
+notably slow and lacks the advanced detection capabilities of Komac.
+
+|                                          |   Komac    | WinGetCreate |                           YamlCreate                           |
+|------------------------------------------|:----------:|:------------:|:--------------------------------------------------------------:|
+| Parameters                               |     ✅      |      ✅       |                               ❌                                |
+| Works without Git                        |     ✅      |      ✅       |                               ❌                                |
+| Optimised manifest ordering*             |     ✅      |      ❌       |                               ✅                                |
+| Fully cross-platform                     |     ✅      |      ❌       |                            Limited                             |
+| Full MSI value retrieval                 |     ✅      |   Partial    |                            Partial                             |
+| Linux and macOS MSI support              |     ✅      |      ❌       |                               ❌                                |
+| Full MSIX value retrieval                |     ✅      |   Partial    |   Limited - https://github.com/Trenly/winget-pkgs/issues/180   |
+| GitHub value retrieval                   |     ✅      |      ❌       |                               ❌                                |
+| Formatted GitHub release notes retrieval |     ✅      |      ❌       |                               ❌                                |
+| Release date identification              |     ✅      |      ❌       |                               ❌                                |
+| Webpage metadata scraping                |     ✅      |      ❌       |                               ❌                                |
+| Fully standalone (w/o winget-pkgs clone) |     ✅      |      ✅       |                               ❌                                |
+| No telemetry                             |     ✅      |      ❌       |                               ✅                                |
+| Type-safety                              |     ✅      |      ✅       |                               ❌                                |
+| Inno setup detection                     |     ✅      |      ✅       |                               ✅                                |
+| Nullsoft detection                       |     ✅      |      ✅       |                               ✅                                |
```

Comment:
Just as a thought, could we indicate that these concepts (how to implement, etc) were contributed by Komac?